sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -1025,9 +1025,14 @@ abstract class CatalogTestUtils {
 
   def newTable(name: String, db: String): CatalogTable = newTable(name, Some(db))
 
+  def newTable(name: String, db: String, catalog: String): CatalogTable = {
```

Comment:
this won't be accepted by OSS as they use n-part name. Can we just use a single `String` and parse it later?

Reply:
sure. This change can be reverted or updated.